example/linux/installation.md

```
@@ -0,0 +1,240 @@
+- While systemd supports environment variables through EnvironmentFile and Environment directives, using absolute paths in both the service file and script ensures consistent behavior across different environments and makes debugging easier.
+
+### Additional Steps for Troubleshooting
+
+1. Verify Node.js and `tsx` installation:
+
+```bash
+node -v
+tsx -v
+```
+
+2. Ensure MongoDB and Redis are running:
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)